distribution/changelog.txt

```diff
@@ -1,18 +1,24 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: OpenGL renderengine added.
 - Feature: Ability to rotate map elements with the tile inspector.
-- Feature: Add cheat to disable littering.
-- Feature: Add ride console command for diagnostics and changing vehicle type.
+- Feature: Added cheat to disable littering.
+- Feature: Addded ride console command for diagnostics and changing vehicle type.
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
+- Feature: Added Norwegian translation and currency.
```

Comment:
There is nothing about Norwegian currency in that PR. I did however alter the Swedish one slightly.